pyroute2/loader.py

```diff
@@ -26,18 +26,4 @@
 
 
 def init():
-    try:
-        # probe, if the bytearray can be used in struct.unpack_from()
-        struct.unpack_from('I', bytearray((1, 0, 0, 0)), 0)
-    except Exception:
-        if sys.version_info[0] < 3:
-            # monkeypatch for old Python versions
-            log.warning('patching struct.unpack_from()')
-
-            def wrapped(fmt, buf, offset=0):
-                return struct._u_f_orig(fmt, str(buf), offset)
-
-            struct._u_f_orig = struct.unpack_from
-            struct.unpack_from = wrapped
-        else:
-            raise
+    struct.unpack_from('I', bytearray((1, 0, 0, 0)), 0)
```

Comment:
This call was just a probe to raise an exception in older python versions, and to prepare a patch for them.

Since we don't support these versions now, there is no need for this probe at all, this `init()` can be removed completely, and the `loader` module can be dropped.

Let me think a bit, how to that in the best possible way.